### PR TITLE
change hurtbox

### PR DESCRIPTION
**Pull request by advilm**

### CODE REVIEW

**Review by SafeDuck — APPROVED** [review]

**Comment by NovaSagittarii** [comment]
i hate this. this is so hard to fix 🗿

File: character_controller_test.tscn [hunk 1]
```diff
@@ -1733,19 +1733,19 @@ script = ExtResource("3_tib08")
...
 [node name="CollisionShape2D" type="CollisionShape2D" parent="Hurtbox"]
 shape = SubResource("RectangleShape2D_sbxtj")
 
 [node name="HurtboxState" type="Node" parent="Hurtbox"]
 script = ExtResource("4_1c3yk")
 
+[node name="Sprite2D" type="Sprite2D" parent="Hurtbox"]
+position = Vector2(1.41799, -0.401596)
+scale = Vector2(0.285608, 0.28652)
+texture = ExtResource("4_380wk")
```
Comment (NovaSagittarii) [inline character_controller_test.tscn:1745]: Somehow became `4_ppaue` instead

Comment (NovaSagittarii) [reply character_controller_test.tscn:1745]: To make things even more fun, in the modified copy (not original), this resource is now `4_itlr7` instead.

**Comment by NovaSagittarii** [comment]
hey wait a minute...

image

**Comment by NovaSagittarii** [comment]
nvm i just copied it over incorrectly

image

**Pull request merged**